### PR TITLE
GH-790 added new CardViewController

### PR DESCRIPTION
**Pull request by DmitryBespalov**
- Added a new base "Card" view controller. All other card-only view controllers will be its descendants.
- Added OnboardingCreationFee view controller
- FeeRequestView encapsulates card's header for the CreationFee
- AddressDetailView encapsulates card's body for the CreationFee

### CODE REVIEW

**Comment by DmitryBespalov** [comment]
Simulator Screen Shot - iPhone X - 2019-05-23 at 15 46 48

**Comment by DmitryBespalov** [comment]
Simulator Screen Shot - iPhone X - 2019-05-23 at 16 36 55

File: SafeUIKit/SafeAppUI/MainFlow/OnboardingFlow/SetupSafeFlow/NewSafeFlow/OnboardingCreationFeeViewController.swift [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                                                             bundle: Bundle(for: CardViewController.self))
+        return controller
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        embed(view: feeRequestView, inCardSubview: cardHeaderView)
+        embed(view: addressDetailView, inCardSubview: cardBodyView)
+
+        navigationItem.title = LocalizedString("create_safe_title", comment: "Create Safe")
```
Comment (sche) [inline SafeUIKit/SafeAppUI/MainFlow/OnboardingFlow/SetupSafeFlow/NewSafeFlow/OnboardingCreationFeeViewController.swift]: let's use enums for localization strings.

File: SafeUIKit/SafeAppUI/MainFlow/OnboardingFlow/SetupSafeFlow/NewSafeFlow/OnboardingCreationFeeViewController.swift [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+        feeRequestView.feeTextLabel.text = LocalizedString("this_is_your_permanent_address", comment: "This is address")
+        feeRequestView.balanceStackView.isHidden = true
+        feeRequestView.feeAmountLabel.amount = TokenData.Ether.withBalance(nil)
+
+        addressDetailView.headerLabel.isHidden = true
+        addressDetailView.address = "0xf1511FAB6b7347899f51f9db027A32b39caE3910"
+        addressDetailView.shareButton.addTarget(self, action: #selector(share), for: .touchUpInside)
+
+        let feeTokenCode = "OWL"
```
Comment (sche) [inline SafeUIKit/SafeAppUI/MainFlow/OnboardingFlow/SetupSafeFlow/NewSafeFlow/OnboardingCreationFeeViewController.swift:50]: You can use ApplicationServiceRegistry.walletService.feePaymentTokenData.code

**Review by sche — APPROVED** [review]

File: SafeUIKit/SafeAppUI/MainFlow/OnboardingFlow/SetupSafeFlow/NewSafeFlow/OnboardingCreationFeeViewController.swift (hunk 2 above)
Comment (DmitryBespalov) [reply SafeUIKit/SafeAppUI/MainFlow/OnboardingFlow/SetupSafeFlow/NewSafeFlow/OnboardingCreationFeeViewController.swift:50]: This is just a UI at this point.

File: SafeUIKit/SafeAppUI/MainFlow/OnboardingFlow/SetupSafeFlow/NewSafeFlow/OnboardingCreationFeeViewController.swift (hunk 1 above)
Comment (DmitryBespalov) [reply SafeUIKit/SafeAppUI/MainFlow/OnboardingFlow/SetupSafeFlow/NewSafeFlow/OnboardingCreationFeeViewController.swift]: 🆗

**Pull request merged**